js/README.md

```diff
@@ -212,6 +212,7 @@ Full list of broader Apache Arrow [projects & organizations](https://github.com/
 
 * [Apache Arrow](https://arrow.apache.org) -- Parent project for Powering Columnar In-Memory Analytics, including affiliated open source projects
 * [rxjs-mapd](https://github.com/graphistry/rxjs-mapd) -- A MapD Core node-driver that returns query results as Arrow columns
+* [Perspective](https://github.com/jpmorganchase/perspective) -- Spinning out of JP Morgan Chase, Perspective is a streaming data visualization engine for Javascript that makes it simple to build real-time & user configurable analytics entirely in the browser.
```

Comment:
I think it would be OK to have this as an example use case for the JS library (vs. solely on Powered By), minus the editorial langage (e.g. "makes it simple" is subjective). What does "spinning out" mean here? We should try to limit name dropping companies (e.g. you won't see Cloudera mentioned in the README for Apache Kudu or Apache Impala, even though most of the devs works there) outside of Powered By